auth/auth.go

```diff
@@ -177,6 +177,17 @@ func (c *Client) CustomTokenWithClaims(uid string, devClaims map[string]interfac
 	return encodeToken(c.snr, defaultHeader(), payload)
 }
 
+// RevokeRefreshToken revokes all refresh tokens for the specified user identified by the uid provided.
+// In addition to revoking all refresh tokens for a user, all ID tokens issued
+// before revocation will also be revoked on the Auth backend. Any request with an
+// ID token generated before revocation will be rejected with a token expired error.
+// Note that due to the fact that the timestamp is stored in seconds, any tokens minted in
```

Comment:
Low level implementation detail. No need to mention it here IMO.

Reply:
I feel that it should be mentioned somewhere. it would be a hard to find error if it just happened. especially as we are returning milliseconds.